### ISSUE TITLE
tfx-cli 0.3.29 - Cannot bind argument to parameter 'Message'

### ISSUE THREAD

**Issue by lemoinek** [jessehouwing/vsts-extension-tasks#27]
Since the update of tfx-cli from 0.3.28 to 0.3.29 I receive this error:

```
##[error]System.Exception: Unexpected exit code 1 returned from tool tfx.cmd
##[error]   at Microsoft.TeamFoundation.DistributedTask.Task.Internal.PowerShell.InvokeToolCmdlet.ProcessRecord()
##[error]   at System.Management.Automation.CommandProcessor.ProcessRecord()
##[error]System.Management.Automation.ParameterBindingValidationException: Cannot bind argument to parameter 'Message' because it is null.
##[error]   at System.Management.Automation.Internal.PipelineProcessor.SynchronousExecuteEnumerate(Object input, Hashtable errorResults, Boolean enumerate)
##[error]   at System.Management.Automation.PipelineOps.InvokePipeline(Object input, Boolean ignoreInput, CommandParameterInternal[][] pipeElements, CommandBaseAst[] pipeElementAsts, CommandRedirection[][] commandRedirections, FunctionContext funcContext)
##[error]   at System.Management.Automation.Interpreter.ActionCallInstruction`6.Run(InterpretedFrame frame)
##[error]   at System.Management.Automation.Interpreter.EnterTryCatchFinallyInstruction.Run(InterpretedFrame frame)
```

**Comment by marag** [comment]
The same problem.

**Comment by lemoinek** [comment]
it seemed to be a problem with the tfx-cli version 0.3.29. 
Microsoft released version 0.3.30 and the bug is fixed.
=> issue closed

**Comment by jessehouwing** [comment]
Thanks for confirming.
